docs/layout-of-source-files.rst

```diff
@@ -276,7 +276,7 @@ functions, annotate conditions for formal verification, and provide a
 function.
 
 In the following example we document the title of the contract, the explanation
-for the two input parameters and two returned values.
+for the two function parameters and two return types.
```

Comment:
`types` does not seem right here. I think we should either say `return parameters` or `return values` (which is slightly wrong, but perhaps better to understand).